store/postgres/src/block_range.rs

```diff
@@ -4,13 +4,28 @@ use diesel::result::QueryResult;
 ///! Utilities to deal with block numbers and block ranges
 use diesel::serialize::{Output, ToSql};
 use diesel::sql_types::{Integer, Range};
+use lazy_static::lazy_static;
+use std::env;
 use std::io::Write;
 use std::ops::{Bound, RangeBounds, RangeFrom};
 
-use graph::prelude::BlockNumber;
+use graph::prelude::{BlockNumber, BLOCK_NUMBER_MAX};
 
 use crate::history_event::HistoryEvent;
 
+lazy_static! {
+    // Make it possible to turn off the expanded BlockRangeContainsClause
+    // if that causes problems in the hosted service. Remove this variable
+    // once we know that it does not lead to performance issues
+    // Setting this to anything will disable the expanded BlockRangeContainsClause
+    static ref DISABLE_BRIN_BLOCK_RANGE: bool = {
+        env::var("DISABLE_BRIN_BLOCK_RANGE")
+            .ok()
+            .map(|_| false)
+            .unwrap_or(true)
```

Comment:
Isn't this backwards, disabling by default and enabling when the variable is present? I think this should be just `ok().is_some()`.

Reply:
Ugh .. yes, you are right. Thanks for spotting that!